code/game/objects/items/storage/backpack.dm

```diff
@@ -71,6 +71,12 @@
 
 /obj/item/storage/backpack/holding/handle_item_insertion(obj/item/W, prevent_warning = 0, mob/user)
 	if((istype(W, /obj/item/storage/backpack/holding) || count_by_type(W.GetAllContents(), /obj/item/storage/backpack/holding)))
+		var/turf/loccheck = get_turf(src)
+		if(loccheck.z == ZLEVEL_CITYOFCOGS)
+			 to_chat(user, "<span class='inathneq_large'><i>I don't think so...</i></span>"
+			 user.visible_message("<span class='inathneq_small'>An unseen force knocks [user] to the ground!</span>")

```

Comment:
remove like 76 and change this one to `user.visible_message("<span class='warning'>An unseen force knocks [user] to the ground!</span>", "<span class='big_brass'>\"What do you think you're doing?\"</span>")`
ratvar uses gold spans, not blue, and visible 'a bad thing has happened to someone' messages use the warning span.

Reply:
yes, only ratvar bitch slaps people, so joan's comment is much more canon.